tox.ini

```diff
@@ -61,23 +62,30 @@ deps =
     -r requirements-lint.txt
 description = Check code against coding style standards
 
-[testenv:kubeflow]
+[testenv:kubeflow-{local,remote}]
```

Comment:
let's track to send a pr to [`bundle-kubeflow`](https://github.com/canonical/bundle-kubeflow/blob/main/.github/workflows/full-bundle-tests.yaml#L163) to run the tests with the new tox env name

Reply:
https://github.com/canonical/bundle-kubeflow/pull/858